addons/mail/static/src/core/public_web/messaging_menu.xml

```diff
@@ -32,7 +32,13 @@
                         <t t-else="" t-esc="thread.displayName"/>
                     </t>
                     <t t-set-slot="body">
-                        <t t-esc="message?.inlineBody or message?.subtype_description" t-att-class="{ 'opacity-75': message?.isEmpty }"/>
+                        <t t-if="message?.hasOnlyAttachments">
+                            <i class="fa me-1" t-att-class="message.messagePreviewIcon"/>
+                            <t t-out="message.messagePreviewText" />
+                        </t>
+                        <t t-else="">
+                            <t t-esc="message?.inlineBody or message?.subtype_description" t-att-class="{ 'opacity-75': message?.isEmpty }"/>
+                        </t>
```

Comment:
Was already there before this PR: this should be a `<span>` rather than `t`: `t-att-class` on text node has no effect, so the empty message text has not the intended reduced opacity.

This could also be in a single line like in previous comment:
```xml
<span t-else="" t-esc="" t-att-class=""/>
```